config/environments/development.rb

```diff
@@ -33,6 +33,8 @@
 
   config.action_mailer.perform_caching = false
 
+  config.action_mailer.default_url_options = { :host => "localhost:3000" }
```

Comment:
I feel like i should also add action mailer configs to production.rb.
What would be the default host?

Reply:
I think we don't have a comarev domain yet, so we can add this in a future pr. @belgamo do we have?